### ISSUE TITLE
Missing out of bounds check?

### ISSUE THREAD

**Issue by dufferzafar** [captn3m0/quick-list-select#1]
This is what happens when you keep going beyond the total items you've entered:

![image](https://user-images.githubusercontent.com/1449512/52418619-6e37c600-2b14-11e9-83c3-b15990f96841.png)

**Comment by captn3m0** [comment]
Yeah, this was a horrible one-time hack. Will try to fix it when I get time
